app/feature/feature-payments/src/main/kotlin/com/hedvig/android/feature/payments/discounts/DiscountsDestination.kt

```diff
@@ -221,12 +221,11 @@ private fun ForeverSection(
     HorizontalItemsWithMaximumSpaceTaken(
       startSlot = {
         HedvigCard(
-          shape = MaterialTheme.shapes.squircleSmall,
           colors = CardDefaults.cardColors(
             containerColor = MaterialTheme.colorScheme.secondaryContainedButtonContainer,
             contentColor = MaterialTheme.colorScheme.onSecondaryContainedButtonContainer,
           ),
-          modifier = Modifier.wrapContentSize(Alignment.TopStart).clickable {
+          modifier = Modifier.wrapContentSize(Alignment.TopStart).clip(MaterialTheme.shapes.squircleSmall).clickable {
```

Comment:
HedvigCard has an overload which takes in a onClick https://github.com/HedvigInsurance/android/blob/a93b5fe072e9aafe0f516c2409d136e16c988098/app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/card/HedvigCard.kt#L50 which is applied at the right place relevant to the clip always I think.
Could you take a look if you can use that one instead here? that way you won't have to specify some shape either, it can just use the default one